modules/default.nix

```diff
@@ -1,4 +1,5 @@
 {
   developer = import ./developer.nix;
   scl-singlenode = import ./scl-singlenode.nix;
```

Comment:
Do we want to export this module as our public interface? If not, we should move this somewhere into `modules/internal/scl-singlenode.nix`. But we can do this in a subsequent PR.